tests/net/igmp/testcase.yaml

```diff
@@ -1,6 +1,7 @@
 common:
   tags: net igmp
   depends_on: netif
+  platform_allow: qemu_cortex_m3 qemu_x86
```

Comment:
Using `qemu_cortex_m3` in LwM2M tests caused some problems with the CI, won't we hit the same issue here? For reference: https://github.com/zephyrproject-rtos/zephyr/pull/54062

Reply:
good question. I had done this same thing a while back for a different test. I can remove the qemu_cortex_m3